javascript-timers/timeout.js

```diff
@@ -0,0 +1,6 @@
+function changeText() {
+  const $h1 = document.querySelector('h1');
```

Comment:
`querySelector`s should be done globally rather than inside functions. By including it here, you're forcing us to traverse the DOM tree over and over every-time this function gets called